commit 5b2ff215473e0526b5b24aeff4ad90d369b21c75
Author: Julien Isorce <julien.isorce@chromium.org>
Date:   Wed Feb 5 17:59:59 2020 +0000

    Fix vaapi with GLX
    
    The signature of ui's gl::GLImageGLX has changed a little bit
    since "mplement GpuMemoryBuffers for EGL and GLX":
      https://chromium-review.googlesource.com/c/chromium/src/+/1984712
    
    Bug: 1031269
    Test: build with use_vaapi=true and run with --use-gl=desktop, see
    Change-Id: I80b07294b9abdfa8233aaf79f7d9ec4c58117090
    https: //chromium.googlesource.com/chromium/src.git/+/refs/heads/master/docs/gpu/vaapi.md#vaapi-on-linux
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2036494
    Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
    Reviewed-by: Miguel Casas <mcasas@chromium.org>
    Commit-Queue: Julien Isorce <julien.isorce@chromium.org>
    Cr-Commit-Position: refs/heads/master@{#738595}

Index: beta/media/gpu/vaapi/vaapi_picture_tfp.cc
===================================================================
--- beta.orig/media/gpu/vaapi/vaapi_picture_tfp.cc
+++ beta/media/gpu/vaapi/vaapi_picture_tfp.cc
@@ -57,7 +57,7 @@ bool VaapiTFPPicture::Initialize() {
   if (make_context_current_cb_ && !make_context_current_cb_.Run())
     return false;
 
-  glx_image_ = new gl::GLImageGLX(size_, GL_RGB);
+  glx_image_ = new gl::GLImageGLX(size_, gfx::BufferFormat::BGRX_8888);
   if (!glx_image_->Initialize(x_pixmap_)) {
     // x_pixmap_ will be freed in the destructor.
     DLOG(ERROR) << "Failed creating a GLX Pixmap for TFP";
